bin/run-all

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+# Test IDs are important because thats the order in which sonobuoy will aggregate results
+
+bin/go-test lifecycle 01
+
+# Add tests here
+bin/go-test inject 02
+bin/go-test ingress 03
```

Comment:
After this test failed on my local cluster, the test script went on to delete my Linkerd control plane. I am not sure how to debug why the tests failed. Any thoughts?

```
$ bin/run-all
....
=== RUN   TestIngress
Running Suite: ingress
======================
Random Seed: 1594265401
Will run 1 of 1 specs

STEP: Installing emojivoto
STEP: Injecting emojivoto
linkerd
  can work with nginx ingress controller
  /home/isim/workspace/linkerd/linkerd2-conformance/tests/ingress/specs.go:18
STEP: Creating ingress-nginx controller
STEP: Injecting linkerd into the ingress controller pods

• Failure [65.663 seconds]
linkerd
/home/isim/workspace/linkerd/linkerd2-conformance/tests/ingress/specs.go:8
  can work with nginx ingress controller [It]
  /home/isim/workspace/linkerd/linkerd2-conformance/tests/ingress/specs.go:18

  could not find a proxy container for deploy/ingress-nginx-controller
  Expected
      <*v1.Container | 0x0>: nil
  not to be nil
...

# went on to run the uninstall test
```

When the script exits, my cluster looks like this:
```
$ k get ns
NAME                 STATUS   AGE
default              Active   31h
ingress-nginx        Active   7m36s  # failed test namespace was left behind, but everything else was deleted
kube-node-lease      Active   31h
kube-public          Active   31h
kube-system          Active   31h
local-path-storage   Active   31h

$ k -n ingress-nginx get po
NAME                                        READY   STATUS      RESTARTS   AGE
ingress-nginx-admission-create-xzhnb        0/1     Completed   0          7m34s
ingress-nginx-admission-patch-wssr4         0/1     Completed   2          7m34s
ingress-nginx-controller-67454c7568-hffz8   2/2     Running     0          6m39s
```

Reply:
Yep, if you want to be able debug your control plane, you need to have a "universal" control plane installation (set `lifecycle.reinstall` to `false`), and skip the uninstall test (set `lifecycle.uninstall` to false). That way your control plane will stay :smile:

Reply:
How about a `fail-fast` option where I can decide to skip the rest of the test cases (and don't destroy my control plane), if one fails? I think `go test -failfast` should work?